shared/js/search_providers.json

```diff
@@ -0,0 +1,24 @@
+{
+  "default": {
+    "google" : {
+      "title": "Google",
+      "searchUrl": "https://www.google.com/search?q={searchTerms}",
+      "suggestUrl": "https://www.google.com/complete/search?client=firefox&q={searchTerms}"
+    },
+    "yahoo": {
+      "title": "Yahoo",
+      "searchUrl": "https://search.yahoo.com/search?p={searchTerms}",
+      "suggestUrl": "https://search.yahoo.com/sugg/ff?command={searchTerms}"
```

Comment:
That will be done in https://bugzilla.mozilla.org/show_bug.cgi?id=1125131, for now I just left the values as they are